### PR TITLE
Larry scep 423

### PR DESCRIPTION
**Pull request by lciummoVZW**
Add capability to example_simple to use file-based certificates.   This removes the capability to use header file certificates.

Cert/keys must still be download from thinspace onto the pi, but you must then run
./scripts/install_file_certs to put the scripts into the store directory.   It takes the same params as the old script that made .h headers but there is NO destination. That's fixed.

Example

[(larry_scep_423**) $ sudo ./install_file_certs.sh 
This script takes 3 (T h r e e)  params
This script takes 3 (T h r e e)  params
This script takes 3 (T h r e e)  params
This script takes 3 (T h r e e)  params

./install_file_certs.sh <path to ca cert> <path to client certificate> <path to client private key> 
Example (assuming script is being executed from the top level of the project):
./install_file_certs.sh ~/Downloads/thingspaceserver.pem ~/Downloads/357353080059530.cert.pem ~/Downloads/357353080059530.private.key

### CODE REVIEW

File: examples/applications/simple/main.c [hunk 1]
```diff
@@ -214,3 +259,219 @@ int main() {
...
+
+int freeCryptoMemory ()
+{
+#warning "file this in!!!!"
+	return 0;
+}
+
+
+#if 0
+int main(void)
```
Comment (mattmcirvin) [inline examples/applications/simple/main.c]: What is this, an alternate main?

Comment (lciummoVZW) [reply examples/applications/simple/main.c]: need to be removed

**Comment by lciummoVZW** [comment]
Will remove extra code and merge tp 423

**Comment by lciummoVZW** [comment]
Add file-based certs to the sample app. Add a script that does the files setup as well...
Add capability to example_simple to use file-based certificates. This removes the capability to use header file certificates.

Cert/keys must still be download from thinspace onto the pi, but you must then run
./scripts/install_file_certs to put the scripts into the store directory. It takes the same params as the old script that made .h headers but there is NO destination. That's fixed.

Example

[�(larry_scep_423**) $ sudo ./install_file_certs.sh
This script takes 3 (T h r e e) params
This script takes 3 (T h r e e) params
This script takes 3 (T h r e e) params
This script takes 3 (T h r e e) params

./install_file_certs.sh 
Example (assuming script is being executed from the top level of the project):
./install_file_certs.sh ~/Downloads/thingspaceserver.pem ~/Downloads/357353080059530.cert.pem ~/Downloads/357353080059530.private.key

**Comment by lciummoVZW** [comment]
shouldnt have close

**Comment by lciummoVZW** [comment]
open for the merge

Add capability to example_simple to use file-based certificates. This removes the capability to use header file certificates.

Cert/keys must still be download from thinspace onto the pi, but you must then run
./scripts/install_file_certs to put the scripts into the store directory. It takes the same params as the old script that made .h headers but there is NO destination. That's fixed.

Example

[�(larry_scep_423**) $ sudo ./install_file_certs.sh
This script takes 3 (T h r e e) params
This script takes 3 (T h r e e) params
This script takes 3 (T h r e e) params
This script takes 3 (T h r e e) params

./install_file_certs.sh 
Example (assuming script is being executed from the top level of the project):
./install_file_certs.sh ~/Downloads/thingspaceserver.pem ~/Downloads/357353080059530.cert.pem ~/Downloads/357353080059530.private.key

**Pull request merged**